features/facebookImport/src/components/dataViz/barChart.css

```diff
@@ -142,7 +142,7 @@
 .bar-chart .bar-box .above-bar p {
     font-size: 14px;
     line-height: 150%;
-    margin: 4px 4px 0 0;
+    margin: 12px 4px 0 0;
```

Comment:
This is probably not part of the PR proper, right?

Reply:
This need first #452 merged. This is a diff with just the changes from this PR: https://github.com//polypoly-eu/polyPod/compare/PROD4POD-1005/Off-fb-fallback...PROD4POD-1005/log-story-variation